src/java/org/apache/cassandra/cql3/statements/CreateTableStatement.java

```diff
@@ -342,8 +344,13 @@ public ParsedStatement.Prepared prepare(Types udts) throws RequestValidationExce
             // If we give a clustering order, we must explicitly do so for all aliases and in the order of the PK
             if (!properties.definedOrdering.isEmpty())
             {
-                if (properties.definedOrdering.size() > columnAliases.size())
-                    throw new InvalidRequestException("Only clustering key columns can be defined in CLUSTERING ORDER directive");
+                List<ColumnIdentifier> nonClusterColumn = properties.definedOrdering.keySet().stream()
```

Comment:
I think we can change the code format to :
`List<ColumnIdentifier> nonClusterColumn = properties.definedOrdering.keySet().stream()
                                                                                                                                           .filter((id) -> !columnAliases.contains(id))
                                                                                                                                           .collect(Collectors.toList());`
which align “.stream()” and ".filter()", ".collect()"

Reply:
Good point! Already changed in the new commit!